code/AEPSDK.brs

```diff
@@ -36,17 +36,40 @@ end function
 
 ' *****************************************************************************
 '
-' Initialize the Adobe SDK and return the public API instance.
-' The following variables are reserved to hold SDK instances in GetGlobalAA():
+' The AEP task node performs the core logic of the SDK.
+' If the task node instance is not provided, calling this function will create a
```

Comment:
Let's reword to something like this:

TaskNode should not provided while initializing SDK if you plan to use this in the MainScene and don't need to create multiple instances of the SDK. If you need to create instances for different screen then you can use getTaskNode API to get the taskNode in the MainScene and use that taskNode to create SDK instance in the other scenes.